lecture_02_data_structures_functions/hw/hw1.py

```diff
@@ -0,0 +1,29 @@
+"""
+Given a file containing text. Complete using only default collections:
+    1) Find 10 longest words consisting from largest amount of unique symbols
+    2) Find rarest symbol for document
+    3) Count every punctuation char
+    4) Count every non ascii char
+    5) Find most common non ascii char for document
+"""
+from typing import List
+
+
+def get_longest_diverse_words(file_path: str) -> List[str, 10]:
```

Comment:
I haven't found any mention of such a size declaration in typing.List. 

```suggestion
def get_longest_diverse_words(file_path: str) -> List[str]:
```